src/main/groovy/org/transmartproject/batch/support/StringUtils.groovy

```diff
@@ -0,0 +1,11 @@
+package org.transmartproject.batch.support
+
+/**
+ * Misc string utils
+ */
+class StringUtils {
+
+    static String escapeForLike(String s, char escapeChar = '\\') {
+        s.replaceAll(/[\\_%]/, "$escapeChar\\\$0")
+    }
```

Comment:
I am using it, especially because if I don't there's no compatibility between Oracle (with assumes no escape character) and Postgres (whwich assumes escape is `\` if none is provided).  What this function does is, assuming the escape character is `\`, escape a literal so that `_` and `%` have no special meaning.

I do see a bug here, in that what I should be replacing is not `[\\_%]`, but `[${escapeChar}_%]`.